README.md

```diff
@@ -12,27 +12,20 @@
   limitations under the License.
 -->
 
-[![Build Status][ti]][tl] [![Maven Central][mi]][ml] [![Issues][ii]][il]
-[![Forks][fi]][fl] [![Stars][si]][sl] [![Apache License][li]][ll]
+[![Build Status][ti]][tl] [![Maven Central][mi]][ml] [![Apache License][li]][ll]
 
-# Fluo.io Parent POM
+# Fluo and Fluo Recipes Parent POM
```

Comment:
I would just call this "Apache Fluo Parent POM" to match the `<name>` in the pom.xml
This will also make maven site documentation, and javadocs, and anything else consistent, if we start using any of those.